tests/Util/yaml_fixtures/add_item_on_config_with_bind.test

```diff
@@ -0,0 +1,28 @@
+services:
+  _defaults:
+    autowire: true
+    autoconfigure: true
+    bind:
+      $httpClient: '@http.client'
+
+  app.service:
+    class: App\Services\Service
+    arguments:
+      - argument_1: 'something'
+        argument_2: 'something'
+===
+$data['services']['app.service']['arguments']['argument_3'] = "something";
+===
+services:
+  _defaults:
+    autowire: true
+    autoconfigure: true
+    bind:
+      $httpClient: '@http.client'
+
+  app.service:
+    class: App\Services\Service
+    arguments:
+      - argument_1: 'something'
+        argument_2: 'something'
+        argument_3: 'something'
```

Comment:
One more thing. The bug you're fixing is related to the `$httpClient` above. Unfortunately, this array syntax down here (while totally legal) is, I think, not yet supported by `YamlSourceManipulator`. And so, the test is STILL failing. I'd recommend updating this test to not use this syntax - instead use:

```yaml
  app.service:
    class: App\Services\Service
    arguments:
        argument_1: 'something'
        argument_2: 'something'
        argument_3: 'something'
```

If you're interested, you could open a separate PR with a failing test case for this array syntax :)

Reply:
Yes, I see Thank you, I just tried somethings and change it how you propose

Reply:
@weaverryan Can you take a look again, please

Thank you